src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -212,12 +222,20 @@ export default {
           this.processing = true
           await this.recoverTransactionAmount()
           this.responseEstimatedGas = await this.getEstimatedGasPrice(this.amountInWei)
-          // TODO: check if response is not null otherwise show an error
+
+          if (!this.responseEstimatedGas) {
+            this.errorMessage = "Couldn't estimate gas fee for swap, please try again soon!"
+            return
+          }
 
           const estimatedGas = new BigNumber(this.responseEstimatedGas?.amount)
             .shiftedBy(-8)
             .toString()
 
+          const swap_balance_proxy_v1 = await this.sharedState.web3.eth.getBalance(
```

Comment:
if you created the config as mentioned above, this would be simply: 
`this.sharedState.web3.eth.getBalance(this.sharedState.currentConfig.swapAddress)`